packages/cli/src/upload/upload.js

```diff
@@ -159,7 +176,7 @@ function getGitHubContext(urlLabel, options) {
 async function runGithubStatusCheck(options, targetUrlMap) {
   const hash = getCurrentHash();
   const slug = getGitHubRepoSlug();
-  const {githubToken, githubAppToken} = options;
+  const {githubToken, githubAppToken, githubApiHost} = options;
```

Comment:
The `getGitHubRepoSlug()` still returned undefined because the `githubApiHost` parameter wasn't passed in, I still got the `No GitHub remote found, skipping.` message in the console. 

Making this change made it work:

```suggestion
  const {githubToken, githubAppToken, githubApiHost} = options;
  const hash = getCurrentHash();
  const slug = getGitHubRepoSlug(githubApiHost);
```

I noticed the `urlLabel` in the status check message is undefined, probably unrelated to this, but posting here:

![image](https://user-images.githubusercontent.com/5751787/76651604-168f3980-6522-11ea-9b14-a4e6f6f02015.png)

Reply:
doh! thanks @jalerg! at some point we just need to get a mock github API into our test suite.

Reply:
everything worked as expected when making this change, I've run it a few times now, and changed the token to our service account instead of mine and it updated the owner smoothly.

LGTM!